src/parser_serial.jl

```diff
@@ -1,3 +1,18 @@
+"""
+    parse_file_serial(
```

Comment:
is there a reason this isn't in the README?

Reply:
I just didn't want to overwhelm readers with too much information. Should I mention it?